xbmc/network/linux/NetworkLinux.cpp

```diff
@@ -20,14 +20,27 @@
 
 #include <cstdlib>
 
+#include "xbmc/ApplicationMessenger.h"
```

Comment:
#include "xbmc/messaging/ApplicationMessenger.h"

Changed in https://github.com/xbmc/xbmc/pull/7268